apollo/agent/agent.py

```diff
@@ -457,6 +462,14 @@ def _execute_client_operation(
                 )
                 url = storage_client.generate_presigned_url(key, expiration_seconds)
                 response.use_location(url)
+                logger.info(
+                    f"Generated pre-signed url for operation: {connection_type}/{operation_name}",
```

Comment:
Logging here as generating pre-signed URL seems significant enough, as it expands the lifetime of the returned data and makes it available to others.